### DISCUSSION TITLE
gator verify -t documentation on output

### DISCUSSION THREAD

**Discussion by KalebDixon11** [open-policy-agent/community#644] [Gatekeeper]
Hi all, we are using gator suites and the gator verify command to test our policies. When you use the `—trace` flag, it shows the trace for the underlying constraint framework evaluation. The constraint framework evaluation has many commands, and I have been searching trying to find documentation on what each of these commands means. Some of the commands are Enter, Eval, Unify, Index, Redo, and Fail. Does anyone know where I can find the documentation stating what these commands are doing during constraint framework evaluation?

**Comment by JaydipGabani** [comment]
@KalebDixon11  You can find more details about these commands here - https://pkg.go.dev/github.com/open-policy-agent/opa/topdown#Op
